pkg/argsbuilder/submit_deepspeedjob.go

```diff
@@ -148,7 +152,9 @@ func (s *SubmitDeepSpeedJobArgsBuilder) check() error {
 
 func (s *SubmitDeepSpeedJobArgsBuilder) setEnv() error {
 	// avoid deepspeed job handing
-	s.args.Envs["NCCL_ASYNC_ERROR_HANDLING"] = "1"
+	if _, ok := s.args.Envs["NCCL_ASYNC_ERROR_HANDLING"]; !ok {
```

Comment:
Use constant to avoid duplication of NCCL_ASYNC_ERROR_HANDLING .

Reply:
Done